.gitignore

```
@@ -25,3 +25,5 @@ exports
 
 # TODO rename the Sanity locale exports to en/nl.json and ignore those files instead
 packages/app/src/locale/*_export.json
+/packages/app/src/locale/en_stripped.json
+/packages/app/src/locale/nl_stripped.json
```

nit; you could make it the same as the line above

```suggestion
packages/app/src/locale/*_stripped.json
```